app/Http/Controllers/ServicesController.php

```diff
@@ -16,7 +16,8 @@ class ServicesController extends Controller
      */
     public function index(Request $request)
     {
-        return Service::where('label', 'like', '%' . $request->get('label', '') . '%')->get();
+        return Service::where('label', 'like', '%' . $request->get('label', '') . '%')
+            ->where('uri', 'like', '%' . $request->get('uri', '') . '%')->get();
```

Comment:
Check with PL or this logic is for a search function or a filter function
Since this URI is an openDataID I think this must be an exact match for only 1 result (but I can be wrong)